### PR TITLE
Clear `top` when not sticking headers to the top

### PR DESCRIPTION
**Pull request by turt2live**
Fixes https://github.com/vector-im/riot-web/issues/14070
For https://github.com/vector-im/riot-web/issues/13635

### CODE REVIEW

File: src/components/structures/LeftPanel2.tsx [hunk 1]
```diff
@@ -119,6 +120,7 @@ export default class LeftPanel2 extends React.Component<IProps, IState> {
                 header.classList.remove("mx_RoomSublist2_headerContainer_stickyTop");
                 header.classList.remove("mx_RoomSublist2_headerContainer_stickyBottom");
                 header.style.width = `unset`;
```
Comment (t3chguy) [inline src/components/structures/LeftPanel2.tsx:122]: consistency :((

Comment (turt2live) [reply src/components/structures/LeftPanel2.tsx:122]: no u

File: src/components/structures/LeftPanel2.tsx [hunk 2]
```diff
@@ -108,6 +108,7 @@ export default class LeftPanel2 extends React.Component<IProps, IState> {
                 header.classList.add("mx_RoomSublist2_headerContainer_sticky");
                 header.classList.add("mx_RoomSublist2_headerContainer_stickyBottom");
                 header.style.width = `${headerStickyWidth}px`;
+                header.style.top = "unset";
```
Comment (turt2live) [inline src/components/structures/LeftPanel2.tsx:111]:
```suggestion
                header.style.top = `unset`;
```

File: src/components/structures/LeftPanel2.tsx [hunk 3]
```diff
@@ -119,6 +120,7 @@ export default class LeftPanel2 extends React.Component<IProps, IState> {
                 header.classList.remove("mx_RoomSublist2_headerContainer_stickyTop");
                 header.classList.remove("mx_RoomSublist2_headerContainer_stickyBottom");
                 header.style.width = `unset`;
+                header.style.top = "unset";
```
Comment (turt2live) [inline src/components/structures/LeftPanel2.tsx:123]:
```suggestion
                header.style.top = `unset`;
```

**Pull request merged**